### PR TITLE
Blog post on DataPackageR onboarding.

### PR DESCRIPTION
**Pull request by gfinak**
@stefaniebutland 
This is the datapackager onboarding blog post.

### CODE REVIEW

File: content/blog/2018-09-18-datapackager.md [hunk 1]
```diff
@@ -0,0 +1,535 @@
...
+
+- Data packages can be version controlled on github (or other VCS sites), making for easy change tracking and sharing with collaborators. 
+
+- When a manuscript is submitted based on a specific version of a data package, one can make a *github release* and automatically push it to sites like [zenodo](http://zenodo.org) so that it is permanently archived. 
+
+- Consumers of the data package pull analysis-ready data from a central location, ensuring all downstream results rely on consistently processed  data sets.
+
+### Development
+
+The package developed organically over the span of several years, initially as a proof-of-concept, with features bolted on over time. Paul Obrecht kindly contributed the data set autodocumentation code and provided in-house testing. As it gained usage internally, we thougt it might be useful to others, and so began the process of releasing it publicly.
```
[the remark this answers is not in the document]
Comment (gfinak) [reply content/blog/2018-09-18-datapackager.md]: He doesn't have a GitHub or twitter account (that I'm aware of) and unfortunately is no longer with our group.

**Comment by gfinak** [comment]
@stefaniebutland I've made changes and pushed them to the forked roweb2 datapackager branch that was the source of this pull request. I've addressed your comments, and also added a small section on limitations and future work, as well as a few sentences in the intro addressing some comments from members of my group.

**Comment by gfinak** [comment]
Thanks! It was a pleasure to contribute.

**Comment by gfinak** [comment]
Just looking over this again and I think I missed some comments (they were helpfully hidden by GitHub). I'll make the remaining changes tomorrow.

**Comment by stefaniebutland** [comment]
Thanks @gfinak

**Comment by gfinak** [comment]
Glad to help. Let me know if there's anything else.

**Pull request merged**